src/test/scala/iog/psg/cardano/CardanoApiSpec.scala

```diff
@@ -59,7 +61,23 @@ class CardanoApiSpec
   }
 
   "GET /wallets/{walletId}/transactions" should "return wallet's transactions" in {
-    api.listTransactions(wallet.id).executeOrFail().map(_.id) shouldBe Seq(createdTransactionResponse.id)
+    val transactions = api.listTransactions(wallet.id).executeOrFail()
+    transactions.map(_.id) shouldBe transactionsIdsDesc
+  }
+
+  it should "run request with proper params" in {
+    val start = ZonedDateTime.parse("2000-01-01T00:00:00.000Z")
+    val end = ZonedDateTime.parse("2001-01-01T00:00:00.000Z")
+    val transactions = api
+      .listTransactions(
+        wallet.id,
+        start = Some(start),
+        end = Some(end),
+        order = CardanoApi.Order.ascendingOrder,
+        minWithdrawal = Some(100)
+      )
+      .executeOrFail()
+    transactions.map(_.id) shouldBe oldTransactionsIdsAsc
```

Comment:
There's a bug in the test 'shouldBe' should be 'shouldEqual'... if that makes sense.

Reply:
hmm I was checking also those tests in a way that changed order, and shouldBe catched that

Reply:
will check it now again

Reply:
ok there was some enum comparision problem, for some reason it worked before now not ;) so Ive fixed that and if I change ordering in Jpi or scala api test catches it, so shouldBe can stay